### PR TITLE
DOC-2002: Added TINY-9510 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2002

Changes:
* Added TINY-9510 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -71,6 +71,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Fixed adding unnecessary space after a comma and before a quote in several scenarios.
+//#TINY-9510
+
+In previous versions of {productname}, the issue of adding an unnecessary space after a comma and before a quote in several scenarios was caused by an incorrectly written `regexp` rule in the typograf library for the Advanced Typography plugin. This led to the appearance of spaces between commas and quotes.
+
+To fix this, {productname}, updated the `regexp` in the typograf library. As a result, spaces between commas and quotes no longer appear.
```
Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:79]:
```suggestion
=== Fixed adding unnecessary space after a comma and before a quote in several scenarios.
//#TINY-9510

In previous versions of {productname}, the issue of adding an unnecessary space after a comma and before a quote in several scenarios was caused by an incorrectly written Regular Expression rule in the https://github.com/typograf[Typograf] library used by the Advanced Typography plugin. This led to the appearance of spaces between commas and quotes.

To fix this, {productname} updated the Regular Expression rule in the Typograf library. As a result, spaces between commas and quotes no longer appear.
```
Copy-edits.

Also, this isn’t a TinyMCE fix. It is a fix to the [Typography plugin](https://github.com/tinymce/typography/blob/main/CHANGELOG.md).

It goes in the _Accompanying Premium Plugin changes_ section, in an _Advanced Typography_ sub-section.

Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc:79]: @brian-forte Good pickup, ill move the content to the required section

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -30,6 +30,20 @@
...
+
+=== Fixed adding unnecessary space after a comma and before a quote in several scenarios.
+//#TINY-9510
+
+In previous versions of {productname}, the issue of adding an unnecessary space after a comma and before a quote in several scenarios was caused by an incorrectly written Regular Expression rule in the https://github.com/typograf[Typograf] library used by the Advanced Typography plugin. This led to the appearance of spaces between commas and quotes.
+
+To fix this, {productname} updated the Regular Expression rule in the Typograf library. As a result, spaces between commas and quotes no longer appear.
+
+For information on the **Advanced Typography** plugin, see: xref:advanced-typography.adoc[Advanced Typography].
 [[accompanying-premium-plugin-end-of-life-announcement]]
```
Comment (kemister85) [inline modules/ROOT/pages/6.5-release-notes.adoc:47]:
```suggestion
For information on the **Advanced Typography** plugin, see: xref:advanced-typography.adoc[Advanced Typography].


[[accompanying-premium-plugin-end-of-life-announcement]]
```

**Review by yacodes — APPROVED** [review]

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 3]
```diff
@@ -30,6 +30,22 @@
...
+The {productname} 6.5 release includes an accompanying release of the **Advanced Typography** premium plugin.
+
+**Advanced Typography** 1.1.3 includes one fix.
+
+=== Fixed adding unnecessary space after a comma and before a quote in several scenarios.
+//#TINY-9510
+
+In previous versions of {productname}, the issue of adding an unnecessary space after a comma and before a quote in several scenarios was caused by an incorrectly written Regular Expression rule in the https://github.com/typograf[Typograf] library used by the Advanced Typography plugin. This led to the appearance of spaces between commas and quotes.
+
+To fix this, {productname} updated the Regular Expression rule in the Typograf library. As a result, spaces between commas and quotes no longer appear.
```
[the remark this answers is not in the document]
Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc:45]: @brian-forte We might need to update this with a new ticket as this has been merged.

**Pull request merged**